config/rbac/role.yaml

```diff
@@ -7,10 +7,10 @@ metadata:
   name: manager-role
 rules:
 - apiGroups:
-  - awsprovider.k8s.io
```

Comment:
I really wish there was consistent sorting for the generation of this file :joy:

Reply:
It should have been fixed in the latest controller-tools

Reply:
https://github.com/kubernetes-sigs/controller-tools/pull/253